packages/qdrant-loader/src/qdrant_loader/connectors/localfile/connector.py

```
@@ -78,129 +79,157 @@ def set_file_conversion_config(self, file_conversion_config: FileConversionConfi
+            content=content,
+            content_type=content_type,
+            metadata=metadata,
+            source_type="localfile",
+            source=self.config.source,
+            url=f"file://{normalized_path}",
+            is_deleted=False,
+            updated_at=updated_at,
+        )
+
+    async def stream_documents(
+        self, since: datetime | None = None
```

Ruff (0.15.15)

[warning] Unused method argument: `since`

(ARG002)